demos/index.html

```diff
@@ -32,20 +33,19 @@
 
     <body>
 
-        <div id="floec-overviewPanel" class="fl-overviewPanel-container"></div>
+        <div class="main">
 
-        <div id="floec-chartAuthoring"></div>
+            <div class="floe-chartAuthoring-container">
```

Comment:
Can divs at line 36 and 38 be combined into one so the html structure has less layers?